week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -24,21 +24,21 @@ function greaterThan10() {
 
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
+  let sortedLetters = letters.sort();
 
   return sortedLetters;
 }
 
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
+  let sliced = numbers.slice(0, 5);
 
   return sliced;
 }
 
-function get3rdIndex(arr) {
-  let index = 3;
-  let element;
+function get3rdIndex() {
+  let index = ["fruit", "banana", "apple", "strawberry", "raspberry"];
+  let element = index[3];
 
   return element;
 }
```

Comment:
If you look at how `get3rdIndex()` is called in the test below, you'll see the array of strings is provided as a parameter.

How would you change the function so that it uses that parameter?